docs/personal-designs.rst

```diff
@@ -0,0 +1,180 @@
+Building Custom Designs
```

Comment:
I think this file might not be strictly needed as it is quite similar on how to build and run pre-existing examples.

Reply:
Fair point. This file was a little repetitive. I went ahead and cut out a few parts from it and referenced other pieces of the documentation instead of re-stating what was already said.
I kept the Building Custom designs file because I think that having a separate section for personal designs will help make it very clear to users that they can run there own designs through the toolchain (something I was not entirely aware I could do when I first read the documentation). I also think there are some pieces of useful information within this file. That being said I don't think it would be to hard to move that information into already existing sections. Let me know what you think.